pos_hide_banknote_button/static/src/css/pos_hide_banknote_button.css

```diff
@@ -4,26 +4,18 @@
     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 */
 
-.pos .payment-numpad .numpad button[data-action="+10"] {
-    visibility: hidden;
-}
-
-.pos .payment-numpad .numpad button[data-action="+20"] {
-    visibility: hidden;
-}
-
-.pos .payment-numpad .numpad button[data-action="+50"] {
+.pos .payment-numpad .numpad button.mode-button {
     visibility: hidden;
 }
 
 .pos .payment-numpad .numpad {
     border-top: 0px solid;
 }
 
-.pos .payment-numpad .numpad button[data-action="1"],
-.pos .payment-numpad .numpad button[data-action="2"],
-.pos .payment-numpad .numpad button[data-action="3"],
-.pos .payment-numpad .numpad button[data-action="BACKSPACE"] {
+.pos .payment-numpad .numpad button:first-child,
+.pos .payment-numpad .numpad button:nth-child(2),
+.pos .payment-numpad .numpad button:nth-child(3),
+.pos .payment-numpad .numpad button:last-child {
```

Comment:
I think that this new way is less predictive that the previous one. If other OCA modules are adding / altering numpad buttons, it could generate conflicts. 

don't you think ? 

Not sure how fix it thought.

Otherwise, lgtm, tested on runbot.

Reply:
Yes but there is no other class which I can use to apply the css. If you have the otherway let me know about it.